raiden/raiden_service.py

```diff
@@ -561,6 +630,8 @@ def close(self, token_address, partner_address):
             second_transfer,
         )
 
+        return netting_channel
```

Comment:
do we really want to expose the internal object `channel.external_state.netting_channel` here? Who is the intended consumer?